### ISSUE TITLE
Have the option to show habitat types on the maps

### ISSUE THREAD

**Issue by Louis-Backstrom** [bird-team/brisbane-bird-atlas#124]
I think it'd be helpful to have a layer that can be added to the maps (even if just on the website) that shows the habitat types - perhaps by colour. It can be a bit confusing knowing what site is what, because often it's hard to know whether a particular area has been classified as non-remnant or not (ie Kedron Brook Wetlands, Sandy Camp, OCC?) - not to mention many hotspots will have the area spread across multiple habitat types but the actual location of the hotspot is only in one - and showing it on the map would be one way of helping this I think.

Does that sound doable? I'll hibernate this as well because it's not a priority, just an idea.

**Comment by jeffreyhanson** [comment]
Yeah, I think this is a fantastic idea! It would take a bit of time, but I think the "Brisbane's Environment" section really needs something like this to show the spatial distribution of the environments that it talks about.

**Comment by Louis-Backstrom** [comment]
Agreed - I think the website would benefit from an overview map of sorts for sure.
